env.template

```diff
@@ -0,0 +1 @@
+training_data = "/path/to/training/data"
```

Comment:
Generally it's good to use upper-case env vars for clarity. Also, do not leave spaces on either side of the `=` in general (it would not work general setting of environment variables in shells, so better not do it here either, though haven't checked out it this case yet)
```suggestion
TRAINING_DATA_PATH="/path/to/training/data"
```